packages/engine/lib/memory/src/shared_memory/shared_memory.rs

```diff
@@ -48,6 +48,20 @@ impl<'id> MemoryId<'id> {
             format!("shm_{id}")
         }
     }
+
+    /// Clean up generade files created by this MemoryIds prefix.
+    pub fn clean_up<Id: Borrow<Uuid>>(id: Id) -> Result<()> {
+        // TODO: Mac differences in shared_memory
```

Comment:
Can you figure out what needs to be done here

Reply:
That's hard for me to figure out as I'm not on a mac so I can't verify, where the files are living.

Reply:
That's fair, I can do the verification if you try a first-go at implementing it. The main thing is just making sure the glob makes sense considering the bespoke logic we have for creating the shmem_ids, I think that's just making sure we're using the same shortening for the id

Reply:
I verified, that we use the correct (shortened) ID for macOS. Actually, macOS does not store shared memory FDs at `/dev/shm/`. Maybe it's stored somewhere else, maybe it's not stored at all (it's technically not needed). I updated the TODO to reflect this.